cattle/plugins/host_info/network.py

```diff
@@ -0,0 +1,54 @@
+import platform
+import os
+import math
+import re
+import json
+import datetime
+
+from cattle.utils import CadvisorAPIClient
+from cattle import Config
+
+
+class NetworkCollector(object):
+    def __init__(self):
+        self.cadvisor = CadvisorAPIClient(Config.cadvisor_ip(),
+                                          Config.cadvisor_port())
+
+    def key_name(self):
+        return "networkInfo"
+
+    def get_data(self):
+        data = {}
+
+        if platform.system() == 'Linux':
+            data.update(self._get_linux_network_info())
+
+        return data
+
+    def _get_linux_network_info(self):
+        stats = self.cadvisor.get_stats()
+        data = {}
+        start = stats[0]
+        end = stats[len(stats) - 1]
+
+        time_fmt = "%Y-%m-%dT%H:%M:%S"
```

Comment:
can you use [timestamp diff](https://github.com/rancher/python-agent/blob/master/cattle/utils.py#L107)? CAdvisor time stamps caused a lot of headaches. It would also keep the time info consistent.